Includes/Rosetta/PlayMode/Conditions/SelfCondition.hpp

```diff
@@ -203,6 +203,10 @@ class SelfCondition
     //! \return Generated SelfCondition for intended purpose.
     static SelfCondition IsOverloadCard();
 
+    //! SelfCondition wrapper for checking the card has combo.
+    //! \return Generated SelfCondition for intended purpose.
+    static SelfCondition IsComboCard();
```

Comment:
I suggest the name of this method to `HasCombo()` due to it has more readability.
What do you think?

Reply:
Other methods like it are
`IsBattlecryCard()`, `HasDeathrattle()`, `HasReborn()`, `IsOverloadCard()`, etc...

So I think, if an ability is for the time of playing a card, `IsAbilityCard()` can be better.
And, if an ability is for the minions already played (in field), `HasAbility()` can be better.

The ability 'Combo' is the former case. So I choose `IsComboCard()`.

What do you think about it?

Reply:
Agree. Thanks for your opinion. 😄